content/reference/v1beta1/config.md

```diff
@@ -235,9 +239,23 @@ Items can be readonly.
           readonly: true
 ```
 
+When used in conjunction with [RandomString](/reference/template-functions/static-context/#randomstring)
```

Comment:
its more than just random string. i think we should generalize this

Reply:
Updated to make it more generic.